packages/runtime/src/widgets/Row.tsx

```diff
@@ -51,6 +59,7 @@ export const Row: React.FC<FlexboxProps> = (props) => {
         flexDirection: "row",
         flexFlow: "unset",
         flexGrow: "unset",
+        cursor: "pointer",
```

Comment:
This should only be pointer if the action callback exists. Same with Column. Can you also update the GridRow with same behavior?